pygmt/src/blockm.py

```diff
@@ -111,6 +112,15 @@ def blockmean(data=None, x=None, y=None, z=None, outfile=None, **kwargs):
 
     {I}
 
+    summary: str
+        [**m**\|\ **n**\|\ **s**\|\ **w**].
+        Type of summary values calculated by blockmean. 
+        
```

Comment:
Need to remove these whitespaces to let the style checks pass.
```suggestion

```

Reply:
Actually, you need to remove the whitespace, but keep the newline :sweat_smile: